Client/Api/Worker/IJobWorkerBuilderStep1.cs

```diff
@@ -228,6 +228,21 @@ public interface IJobWorkerBuilderStep3
         /// <returns>the builder for this worker</returns>
         IJobWorkerBuilderStep3 AutoCompletion();
 
+        /// <summary>
+        /// Specifies how many handler threads are used by this job worker.
+        /// </summary>
+        ///
+        /// <para>
+        /// The previous defined job handler can be called by multiple threads, to execute more jobs concurrently.
+        /// Per default three job handler threads are used by an job worker.
```

Comment:
I think the default is just one job worker thread. Right?

Reply:
True thanks :+1: